### ISSUE TITLE
Active vs non active buttons, avoid confusion

### ISSUE THREAD

**Issue by infinite-dao** [Human-Connection/WebApp#221]
Hej-hej,

On buttons shout and follow I found them inverse to visualize the active state :

![screenshot buttons active vs active color](https://user-images.githubusercontent.com/17983598/43365314-434704d2-932b-11e8-962a-a9514e447320.png)

So I don’t know, :confused: the more green, the more I can do/engage with or is already active ? 

It may also confusing related to to boolean switches which indicate activeness in green, so, on a button that was activated I recommend a light color variant, in this case a light green variant

Andreas

**Comment by appinteractive** [comment]
I agree! But I do lot have a quick solution here to be honest. Do you? 

Because we also have primary action buttons (no state change) that are also green.
